packages/components/input/src/use-input.ts

```diff
@@ -370,7 +370,11 @@ export function useInput<T extends HTMLInputElement | HTMLTextAreaElement = HTML
         "data-has-start-content": dataAttr(hasStartContent),
         "data-has-end-content": dataAttr(!!endContent),
         className: slots.input({
-          class: clsx(classNames?.input, isFilled ? "is-filled" : ""),
+          class: clsx(
+            classNames?.input,
+            isFilled ? "is-filled" : "",
+            isMultiline ? "px-0 rtl:px-0" : "",
```

Comment:
we don't use `rtl:`. please sync once with canary branch then use `pe-0`.

Reply:
pe-0 doesn't seem to be working，and **rtl:** mainly solves the padding problem in PTL case

Reply:
you need to sync canary branch to your working branch first because your branch is still using `rtl:` so `pe-0` won't work.

Reply:
thanks，I've revise it